src/services/FlawService.ts

```diff
@@ -136,18 +136,21 @@ export async function postFlawCvssScores(flawId: string, cvssScoreObject: unknow
 }
 
 export async function postFlawComment(
-  uuid: string, comment: string, creator: string, isPrivate: boolean, embargoed: boolean
+  flawId: string, comment: string, creator: string, isPrivate: boolean, embargoed: boolean
 ) {
+  const type = isPrivate ? 'Private' : 'Public';
   return osidbFetch({
     method: 'post',
-    url: `/osidb/api/v1/flaws/${uuid}/comments`,
+    url: `/osidb/api/v1/flaws/${flawId}/comments`,
     data: {
       text: comment,
       creator: creator,
       is_private: isPrivate,
       embargoed,
     },
-  }).then((response) => response.data);
+  }).then(createSuccessHandler({ title: 'Success!', body: `${type} comment saved.` }))
+    .then((response) => response.data)
+    .catch(createCatchHandler(`Error saving ${type} comment`));
```

Comment:
I have the same opinion here, but I see we do handle errors on all services of this file, so maybe we should decide if we handle errors inside each service or not, to keep consistency between services